guides/app-guides/app-faq.md

```diff
@@ -49,7 +49,7 @@ In the case there is not enough funds in a pool Hedging Agents are withdrawing f
 
 All collateral/agTokens pools are separated and managed by independent contracts. This means that, even though pools for a given collateral can use the same strategies, it could be the case that they don't or that some parameters related to these strategies differ.
 
-Additionally, the amounts put by users minting on these pools differ, which impact the multiplier effect explained [here](https://docs.angle.money/concepts/standard-liquidity-providers#multiplier-effect), and change the APYs for SLPs.
+Additionally, the amounts put by users minting on these pools differ, which impact the multiplier effect explained [here](/core-module/standard-liquidity-providers/README.md#✖-multiplier-effect), and change the APYs for SLPs.
```

Comment:
Link will be broken here

Reply:
Yes but core-module is not on the same repo: you need to do something like ../../core-module